src/httpd.rs

```diff
@@ -273,6 +367,17 @@ impl Server {
         Ok(())
     }
 
+    fn register_error_handler(&mut self, handler: ErrorHandler) -> Result<()> {
```

Comment:
Is de-registration of these necessary, on server stop?

Reply:
probably, I'll check